python/cudnn/deepseek_sparse_attention/indexer_backward/dense_indexer_backward_sm100.py

```
@@ -326,7 +342,7 @@ class SharedStorage:
 # Barrier indices
 MBAR_2Q_S_FULL = 0  # MMA → Compute (phase-flipped 2×/block)
 MBAR_2Q_DS_READY = 1  # Compute → MMA (phase-flipped 2×/block)
-MBAR_2Q_DK_FULL = 2  # MMA → Reduce (1×/block, after Phase B GEMM2)
+MBAR_2Q_DK_FULL = 2  # MMA → Reduce/Compute (1×/block, after dK/dQ GEMMs)
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)